tests/pipelines/main.nf.test

```diff
@@ -427,4 +435,173 @@ nextflow_pipeline {
             assert iridanext_metadata.isEmpty()
         }
     }
+
+    test("Testing mismatched IDs") {
+        // IDs in the sample sheet and IDs in the individual MLST JSON files will not match.
+        // This tests the pipelines ability to handle and correct for this problem.
+
+        tag "mismatch"
+
+        when {
+            params {
+                input = "$baseDir/tests/data/samplesheets/samplesheet-mismatched-ids.csv"
+                outdir = "results"
+
+                pd_distm = "scaled"
```

Comment:
I wanted to make a note of this, but think it's something that only needs to be looked at prior to a new release, not for this PR.

We are setting `pd_distm` to *scaled* here and the `gm_thresholds` is set to numbers from 0 to 1 (i.e., a proportion). But the distances in the expected distances file and the tree seem to be given in terms of percentages.